src/device/device_ledger.cpp

```diff
@@ -1982,7 +1991,7 @@ namespace hw {
     }
   
   #else //WITH_DEVICE_LEDGER
-  
+    #warning "Ledger Device not enabled"
```

Comment:
I don't think we need this

Reply:
OK (bis) was just added to focus on it is disable. But I can remove it.